src/services/rns/models/expiration.model.ts

```diff
@@ -3,6 +3,9 @@ import { Column, DataType, Model, Table } from 'sequelize-typescript'
 @Table({ freezeTableName: true, tableName: 'rns_domain_expiration', timestamps: false })
 export default class DomainExpiration extends Model {
   @Column({ primaryKey: true, type: DataType.STRING })
+  id!: string
+
+  @Column({ type: DataType.STRING })
   tokenId!: string
```

Comment:
Do I understand it right, that you are duplicating the `tokenId`? This seems like quite lazy solution.